### ISSUE TITLE
Failed with No Reponse Warnings Before HA Core Crash

### ISSUE THREAD

**Issue by Nerwyn** [krahabb/meross_lan#403]
## Version of the custom_component

5.0.1

## Configuration

[config_entry-meross_lan-bad25ae10b5ed89af4f761002dc837e4.json](https://github.com/krahabb/meross_lan/files/14466021/config_entry-meross_lan-bad25ae10b5ed89af4f761002dc837e4.json)

## Describe the bug
There might be an issue with the latest version of this integration causing a HA core to crash. My normally stable Home Assistant OS setup has crashed three times today with the following error as the last one in the logs:

```
msg100_###############################7] Appliance.Control.Multiple failed with no response: requests=2 expected size=1030
```

## Debug log
The first log is from the second crash. The second is after enabling debug logging and up to Home Assistant crashing a third time. While not visible or copyable, the final line is full of `NULL`s like so.
[home-assistant-crash-1.log](https://github.com/krahabb/meross_lan/files/14466077/home-assistant-crash-1.log)
[home-assistant-crash-2.log](https://github.com/krahabb/meross_lan/files/14466081/home-assistant-crash-2.log)
![image](https://github.com/krahabb/meross_lan/assets/43071698/c7c28659-1ce3-4345-abd6-650b7d0944ba)

**Comment by Nerwyn** [comment]
Since downgrading to 5.0.0 Home Assistant hasn't crashed.

**Comment by krahabb** [comment]
Hello @Nerwyn,
I'm always a bit suspicious about a piece of python code able to crash the whole application (at least for HA) since any software error in the component is (in the end) managed by the core framework which should just dump a verbose log about the issue and not crash at all. The only thing that comes to mind, since re-installing a previous version fixed the issue, is that the previous 5.0.1 installation was somewhat corrupted. It happens more than it should and I suspect sometimes the HACS download silently fails with no critical errors and so the component looks correctly installed but the whole file base is not properly in place maybe missing some files or even worse, and likely causing the crash, having some files with polluted data...
I wonder if this could also happen when manually using git to download the repository files or any other utility.

At the moment there are no other reports about v5.0.1 hard-crashes like this, and differences in code between 5.0.0 and 5.0.1 are so tiny and mainly pointed at the RollerShutter and Smart Fan/Air purifier that I'm still convinced the issue was not effectively related to a bug in 5.0.1 but, as explained above, to an 'unfortunate' install.

We'll keep an eye on this, nevertheless.

**Comment by Nerwyn** [comment]
Yup, it happened again twice today despite downgrading. It looks like it's a Zigbee bellows ezsp problem.
